gapis/api/gvr/BUILD.bazel

```diff
@@ -53,6 +53,8 @@ apic_template(
     api = ":api",
     templates = [
         "//gapis/api/templates:api",
+        "//gapis/api/templates:api_types",
+        "//gapis/api/templates:api_box_memory",
```

Comment:
You call this memory_box everywhere else but here.

Reply:
Fixed